scripts/scil_volume_pairwise_comparison.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Evaluate pair-wise similarity measures of masks and atlas.
+All volumes must be in the same space.
+
+For the voxel representation, the computed similarity measures are:
+    adjacency_voxels, dice_voxels, volume_overlap, volume_overreach.
+
+This can work for BET mask, WMPARC, bundle label maps. The datatype of the
+input volumes must be uint8 (mask) or uint16 (label map and atlas).
+"""
+
+
+import argparse
+import itertools
+import json
+import logging
+import multiprocessing
+
+from dipy.io.utils import get_reference_info
+import numpy as np
+
+from scilpy.io.image import load_img
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             add_processes_arg,
+                             add_reference_arg,
+                             add_verbose_arg,
+                             assert_headers_compatible,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             validate_nbr_processes)
+from scilpy.image.labels import get_data_as_labels
+from scilpy.tractanalysis.reproducibility_measures import compute_dice_voxel, \
+    compute_bundle_adjacency_voxel
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_volumes', nargs='+',
+                   help='Path of the input volumes.')
```

Comment:
volume(s)?

Or if there should always be at least two, please add a verification and add it in the explanation, please.

Reply:
Without --single_compare you need at least two, if using --single_compare you can have one (Added to the docstring)